x/gov/client/cli/query.go

```diff
@@ -674,3 +675,47 @@ $ %[1]s query gov param deposit
 	flags.AddQueryFlagsToCmd(cmd)
 	return cmd
 }
+
+// GetCmdCertVoted implements the query param command.
+func GetCmdCertVoted() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "cert-voted [proposa-id]",
+		Args:  cobra.ExactArgs(1),
+		Short: "Query if the certifiers voted on a proposal",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Query if the certifiers voted on a proposal.
+Example:
+$ %[1]s query gov cert-voted 1
+`,
+				version.AppName,
+			),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			queryClient := types.NewQueryClient(cliCtx)
+
+			proposalId, err := strconv.ParseUint(args[0], 10, 64)
```

Comment:
Need to fix this bug reported by golangci-lint?

Reply:
done